### ISSUE TITLE
Change some border width semantic tokens

### ISSUE THREAD

**Issue by julien-deramond** [Orange-OpenSource/ouds-ios#106]
### Description

The references have changed for the border width semantic tokens.
The new semantic border width tokens are:

| Name | Value |
| ---- | ----- |
| border-width-none | border-width-0 |
| border-width-default | border-width-25 |
| border-width-thin | border-width-25 |
| border-width-medium | border-width-50 |
| border-width-thick | border-width-75 |
| border-width-thicker | border-width-100 |
| border-width-outside-focus | border-width-50 |

In summary:
* `border-width-thickest` doesn't exist anymore
* `border-width-medium` is new
* `border-width-interactive-primary-focus` doesn't exist anymore
* `border-width-outside-focus` is new
* `border-width-thick` has a different value
* `border-width-thicker` has a different value

**Comment by pylapp** [comment]
📣 **New TestFlight ALPHA upload** 🚀
- Display name: **OUDS Showcase ALPHA (78, 40, 9, 32, 106, 94, 87, 58, 89)**
- Version: 0.2.0
- Build number: **1726677063**
- Build tag: fcb55ec
- Build type: ALPHA
- Build details (GitHub):78, 40, 9, 32, 106, 94, 87, 58, 89

_This is an automated message sent with *Fastlane* from our CI/CD pipeline 🤘_
